Repository/PackageRepositoryFactory.cs

```diff
@@ -12,27 +12,42 @@ public static PackageRepositoryFactory Default
             get { return _default;}
         }
 
-        public virtual IPackageRepository CreateRepository(string packageSource, NuGetRequest request)
+        public IPackageRepository CreateRepository(PackageRepositoryCreateParameters parms)
```

Comment:
>parms [](start = 85, length = 5)

params?

Reply:
params is a reserved keyword

---
In reply to: [154537885](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154537885) [](ancestors = 154537885)

Reply:
Consider parameters as parms is looking like a typo.